package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@divriots/starter-smelte",
+	"name": "@backlight-dev/divriots.starter-smelte",
```

Comment:
this wasn't in my changeset... strange, I'm gonna have a look why

Reply:
@gluck @gqio  any idea where it could come from when making a branch from Backlight?